packages/@okta/vuepress-site/docs/guides/build-sso-integration/before-you-begin/openidconnect/prep.md

```diff
@@ -9,9 +15,10 @@ Before you create a new OIDC integration in Okta:
    * A single page app (SPA) is a web application that is contained on a single web page. All code is retrieved when the page is loaded initially - the page doesn't reload or refresh. A SPA application cannot keep running on a server.
 -->
 
+1. Have your application developed and tested - with a front-end (for example, JavaScript and HTML) and back-end (for example, middleware and database software) stack, with services available through APIs and accepting HTTP connections.
+1. Based on the [type of application you have built](/docs/concepts/auth-overview/#what-kind-of-client-are-you-building), determine the correct [OAuth 2.0 flow](/docs/concepts/auth-overview/#recommended-flow-by-application-type) that is required below the OIDC identity layer. For OIN app integrations, the OAuth 2.0 flow must be either Web (server side) or a Single Page Application (SPA).
```

Comment:
type of application you have built -> type of application that you have built

for translation

Single Page Application -> Single-Page Application